pkg/network/node/vnids.go

```diff
@@ -208,6 +223,9 @@ func (vmap *nodeVNIDMap) handleAddOrUpdateNetNamespace(obj, _ interface{}, event
 	oldNetID, err := vmap.getVNID(netns.NetName)
 	oldMCEnabled := vmap.mcEnabled[netns.NetName]
 	mcEnabled := netnsIsMulticastEnabled(netns)
+	if vmap.findDuplicateNetID(netns.NetName, netns.NetID) {
```

Comment:
This has no relation to this block of code, which is semantically tied to the comment `// Skip this event if nothing has changed`. Could you please move this before the comment or after?